controllers/storage/provision.go

```diff
@@ -139,16 +178,17 @@ func (m *MinIOReconciler) generateAzureSecret() *corev1.Secret {
 		},
 		Type: corev1.SecretTypeOpaque,
 		Data: map[string][]byte{
```

Comment:
As secret is created, why do we still need to put the related info in the secret data field of the secret property?

Reply:
what do you meaning? 
the secret will be created after putting the related info in the secret, and harbor will get all property in the data field by secret name.